### PR TITLE
config: Add an rpm-ostree prow job

### PR DESCRIPTION
**Pull request by cgwalters**
So we get "it builds and runs" sanity checking.

### CODE REVIEW

**Comment by cgwalters** [comment]
Hm, I don't understand these failures.  This looks like fallout from https://github.com/openshift/release/commit/ef3cd1d0c5a65fc78dc9b4a4d07d536d983554f1

File: ci-operator/config/coreos/rpm-ostree/OWNERS [hunk 1]
```diff
@@ -0,0 +1,14 @@
+approvers:
```
Comment (vrutkovs) [inline ci-operator/config/coreos/rpm-ostree/OWNERS]: These need `reviewers` so that we could LGTM PRs here

**Comment by cgwalters** [comment]
Hooray, this works now!

File: ci-operator/jobs/coreos/rpm-ostree/OWNERS [hunk 2]
```diff
@@ -0,0 +1,14 @@
+approvers:
```
Comment (vrutkovs) [inline ci-operator/jobs/coreos/rpm-ostree/OWNERS]: These need `reviewers` so that we could LGTM PRs here

Comment (cgwalters) [reply ci-operator/jobs/coreos/rpm-ostree/OWNERS]: Aren't approvers also reviewers?

Comment (vrutkovs) [reply ci-operator/jobs/coreos/rpm-ostree/OWNERS]: No, these are different - https://github.com/kubernetes/test-infra/tree/master/prow/plugins/approve/approvers#overview

Comment (cgwalters) [reply ci-operator/jobs/coreos/rpm-ostree/OWNERS]: Hmm but how does e.g. https://github.com/openshift/machine-config-operator/blob/master/OWNERS and https://github.com/openshift/cluster-version-operator/blob/master/OWNERS etc. work then?

Comment (vrutkovs) [reply ci-operator/jobs/coreos/rpm-ostree/OWNERS]: Oh, it seems it works like this too

Comment (cgwalters) [reply ci-operator/jobs/coreos/rpm-ostree/OWNERS]: Can you `/lgtm` this?

File: ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
```
Comment (petr-muller) [inline ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml]: Remove `cluster: `, it's never needed and causes problems.

Comment (cgwalters) [reply ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml]: Done!

File: ci-operator/config/coreos/rpm-ostree/OWNERS (hunk 1 above)
Comment (stevekuznetsov) [reply ci-operator/config/coreos/rpm-ostree/OWNERS]: Approvers get used when no reviewers get listed

File: ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+tests:
+- as: sanity
+  commands: rpm-ostree --version
+  container:
+    from: bin
+
+resources:
+  '*':
+    limits:
+      memory: 6Gi
```
Comment (stevekuznetsov) [inline ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml]: That's a lot of memory for your sanity test :)

Comment (cgwalters) [reply ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml]: Hm yeah, the build process (gcc and Rust compilers) is what uses RAM; could dial that back down some.

Comment (cgwalters) [reply ci-operator/config/coreos/rpm-ostree/coreos-rpm-ostree-master.yaml]: Ultimately I think what we really want is to

1) Attribute resource usage to repositories and organizations and keep it sane
2) https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler

**Pull request merged**